src/Data/Sum/Algebra.agda

```diff
@@ -0,0 +1,64 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Algebraic properties of sums (disjoint unions)
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Sum.Algebra where
+
+open import Agda.Builtin.Sigma
+open import Algebra.Definitions
+open import Data.Empty using (⊥)
+open import Data.Sum.Base
+open import Data.Sum.Properties
+open import Function.Base using (id)
+open import Level
+
+open import Function.Bundles using (_↔_; Inverse; mk↔)
+import Function.Definitions as FuncDef
+
+open import Relation.Binary.PropositionalEquality.Core using (_≡_; refl)
+
+------------------------------------------------------------------------
+
+module _ {a b : Level} {A : Set a} {B : Set b} where
```

Comment:
Using variables for the levels and the later sets and would clear things up a bit I think?

Reply:
I can use variables for the levels, but not the sets, as they are explicitly passed to an `open` statement.